apidocs/feature-flags.md

```diff
@@ -0,0 +1,228 @@
+
+```json
+[
+    { "op": "replace", "path": "/environments/production/on", "value": false }
+]
+```
+
+Each section of the Targeting tab corresponds to a different attribute of the per-environment configuration data, as shown here:
+
+## Individual user targets
+
+The `targets` array in the per-environment configuration data represents the individual user targeting rules on the Targeting tab. Each object in the `targets` array represents a list of user keys assigned to a particular variation. For example:
+
+```json
+{
+    ...
+    "environments" : {
+        "production" : {
+            ...
+            "targets": [
+                {
+                    "values": ["foo@example.com"],
+                    "variation: 0
+                }
+            ]
+        }
+    }
+}
+```
+
+This `targets` array here means that user `foo@example.com` should receive the first variation listed in the `variations` array (recall that the variations are stored at the top level of the flag JSON in an array, and the per-environment configuration rules point to indexes into this array). If this is a boolean flag, that means that `foo@example.com` is receiving the `true` variation.
+
+## Targeting rules
+
+The `rules` array corresponds to the custom targeting rules section of the Targeting tab. This is where you can express complex rules on attributes with conditions and operators. For example, a rule like "roll the `true` variation out to 80% of users whose e-mail address ends with `gmail.com` would be expressed here.
+
+## The fallthrough rule
+
+The `fallthrough` object is a special rule that contains no conditions-- it is the rollout strategy that is applied when none of the individual or custom targeting rules match.
+
+## The off variation
+
+The off variation represents the variation to serve if the feature flag is killed (the `on` attribute is `false`). For boolean flags, this is usually `false`. For multivariate flags, set the off variation to whatever variation represents the control or baseline behavior for your application. If you don't set the off variation, LaunchDarkly will serve the fallback value defined in your code.
+
+## Percentage rollouts
+
+The `weight` attribute defines the percentage rollout for each variation. Weights range from 0 (0% rollout) to 100000 (a 100% rollout). The weights are scaled by a factor of 1000 so that fractions of a percent can be represented without using floating point-- so a weight of `50000` means that 50% of users (that don't otherwise match any `targets`, see above) will see that variation. The sum of weights across all variations should be 100%.
```

Comment:
```suggestion
The `weight` attribute defines the percentage rollout for each variation. Weights range from 0 (a 0% rollout) to 100000 (a 100% rollout). The weights are scaled by a factor of 1000 so that fractions of a percent can be represented without using floating-point. For example, a weight of `50000` means that 50% of users that don't otherwise match any `targets` will see that variation. The sum of weights across all variations should be 100%.
```